pkg/pgmodel/ingestor/trace/writer.go

```diff
@@ -207,6 +213,10 @@ func (t *traceWriterImpl) InsertTraces(ctx context.Context, traces pdata.Traces)
 			}
 		}
 	}
+
+	metrics.InsertBatchSize.With(prometheus.Labels{"type": "trace", "kind": "instrumentation"}).Observe(float64(instrLibBatch.b.Len()))
```

Comment:
I think this is now `instrumentation_lib` as per metric renaming.

```suggestion
	metrics.InsertBatchSize.With(prometheus.Labels{"type": "trace", "kind": "instrumentation_lib"}).Observe(float64(instrLibBatch.b.Len()))
```